Hardware/HardwareTests/PaymentIntentTests.swift

```diff
@@ -0,0 +1,60 @@
+import XCTest
+@testable import Hardware
+
+/// Tests the mapping between PaymentIntent and SCPPaymentIntent
+final class PaymentIntentTests: XCTestCase {
+    func test_intent_maps_id() {
+        let mockIntent = MockStripePaymentIntent.mock()
```

Comment:
nit: it seems like each test case uses the same intent, maybe we could have a private property in this class like `private let mockIntent = MockStripePaymentIntent.mock()` for each test case to use?

Reply:
Interesting. My first impulse was to reply "I would prefer not to, to avoid sharing state between tests" but the mock is immutable so there is no danger of having one test affecting other... Updating!